packages/tests/tests/peer_exchange.node.spec.ts

```diff
@@ -70,16 +75,32 @@ describe("Peer Exchange", () => {
       expect(peerInfos.length).to.be.greaterThan(0);
       expect(peerInfos.length).to.be.lessThanOrEqual(numPeersToRequest);
       expect(peerInfos[0].ENR).to.not.be.null;
+      expect(peerInfos[0].ENR?.peerInfo?.multiaddrs).to.not.be.null;
 
+      let foundNodeMas: Multiaddr[] = [];
```

Comment:
nit: `foundNodeMaddr` is a better name as `mas` is not associated with addresses

Reply:
renamed to `foundNodeMultiaddrs`